### PR TITLE
[OR-1401] add devnet command for eth

### PR DESCRIPTION
**Pull request by rlgns98kr**
- add devnet command for eth

There is a request to enable to test in eth.
So I added devnet command for eth

Thank you! 🙇🏼‍♂️

### CODE REVIEW

**Comment by 0x6e616d** [comment]
I think we can use "legacy" for this case, using "eth" is confused

**Comment by rlgns98kr** [comment]
> I think we can use "legacy" for this case, using "eth" is confused 

I think it is great idea! Let me update it. 🙇🏻‍♂️

File: Makefile [hunk 1]
```diff
@@ -116,9 +116,15 @@ pre-devnet:
...
 		|| make devnet-allocs
-	PYTHONPATH=./bedrock-devnet $(PYTHON) ./bedrock-devnet/main.py --monorepo-dir=.
+	PYTHONPATH=./bedrock-devnet $(PYTHON) ./bedrock-devnet/main.py --monorepo-dir=. --native
 .PHONY: devnet-up
 
+devnet-legacy-up: pre-devnet
+	./ops/scripts/newer-file.sh .devnet/allocs-l1.json ./packages/contracts-bedrock \
+		|| make devnet-allocs
+	PYTHONPATH=./bedrock-devnet $(PYTHON) ./bedrock-devnet/main.py --monorepo-dir=.
+.PHONY: devnet-eth-up
```
Comment (0x6e616d) [inline Makefile:126]:
```suggestion
.PHONY: devnet-legacy-up
```

File: Makefile [hunk 2]
```diff
@@ -138,6 +144,14 @@ devnet-clean:
 	docker volume ls --filter name=ops-bedrock --format='{{.Name}}' | xargs -r docker volume rm
 .PHONY: devnet-clean
 
+devnet-legacy-clean:
+	rm -rf ./packages/contracts-bedrock/deployments/devnetL1
+	rm -rf ./.devnet
+	cd ./ops-bedrock && docker compose down
+	docker image ls 'ops-bedrock*' --format='{{.Repository}}' | xargs -r docker rmi
+	docker volume ls --filter name=ops-bedrock --format='{{.Name}}' | xargs -r docker volume rm
+.PHONY: devnet-clean
```
Comment (0x6e616d) [inline Makefile:153]:
```suggestion
.PHONY: devnet-legacy-clean
```

File: Makefile [hunk 3]
```diff
@@ -116,9 +116,15 @@ pre-devnet:
 devnet-up: pre-devnet
 	./ops/scripts/newer-file.sh .devnet/allocs-l1.json ./packages/contracts-bedrock \
 		|| make devnet-allocs
-	PYTHONPATH=./bedrock-devnet $(PYTHON) ./bedrock-devnet/main.py --monorepo-dir=.
+	PYTHONPATH=./bedrock-devnet $(PYTHON) ./bedrock-devnet/main.py --monorepo-dir=. --native
```
Comment (0x6e616d) [inline Makefile:119]: I think should better if we use `legacy` for this flag, by default is using our contracts

**Comment by rlgns98kr** [comment]
@0x6e616d 
Thank you for your valuable feedback. I updated your idea! I'm sorry for my lots of missing! 🙇🏼‍♂️

File: bedrock-devnet/devnet/__init__.py [hunk 4]
```diff
@@ -55,15 +56,15 @@ def main():
 
     monorepo_dir = os.path.abspath(args.monorepo_dir)
     devnet_dir = pjoin(monorepo_dir, '.devnet')
-    contracts_bedrock_dir = pjoin(monorepo_dir, 'packages', 'tokamak', 'contracts-bedrock')
+    contracts_bedrock_dir = pjoin(monorepo_dir, 'packages', 'contracts-bedrock') if args.legacy else pjoin(monorepo_dir, 'packages', 'tokamak', 'contracts-bedrock')
```
Comment (nguyenzung) [inline bedrock-devnet/devnet/__init__.py:59]: I am not sure if we can use "if not" instead of " if args.legacy else"

**Review by nguyenzung — APPROVED** [review]
Great job, Steven

File: bedrock-devnet/devnet/__init__.py (hunk 4 above)
Comment (rlgns98kr) [reply bedrock-devnet/devnet/__init__.py:59]: In fact, I didn't know Python grammer well. I should study more, I'm sorry. Let me update it! 🙇🏻‍♂️

Comment (nguyenzung) [reply bedrock-devnet/devnet/__init__.py:59]: Oh, i am sorry. I dont know much about Python, Please keep it

Comment (rlgns98kr) [reply bedrock-devnet/devnet/__init__.py:59]: In fact, I want to do like your opinion. 😄
I tried !, but it is not working. 😊 kkk

I learned the correct grammer is the 'not' thanks to you!
I will test and update it tomorrow! 🙇🏻‍♂️

Comment (0x6e616d) [reply bedrock-devnet/devnet/__init__.py:59]: you can use

```suggestion
    if args.legacy:
      contracts_bedrock_dir = pjoin(monorepo_dir, 'packages', 'contracts-bedrock')
    else:
      contracts_bedrock_dir = pjoin(monorepo_dir, 'packages', 'tokamak', 'contracts-bedrock')
```
```

Comment (nguyenzung) [reply bedrock-devnet/devnet/__init__.py:59]: With Python, I like shorter source code more. It is personal opinion. ^^

Comment (rlgns98kr) [reply bedrock-devnet/devnet/__init__.py:59]: I like shorter too. Please forgive me @0x6e616d,,,🥰

**Review by 0x6e616d — APPROVED** [review]
SGTM

**Pull request merged**